revenue_timeseries.yaml

```diff
@@ -43,121 +48,98 @@ tables:
-        synonyms:
-          - absolute error
-          - L1
-        description: absolute error between forecasted and actual revenue
-        expr: abs(forecasted_revenue - revenue)
-        data_type: number
-        default_aggregation: avg
 
-  - name: daily_revenue_by_region
-    description: Daily revenue sliced by region, aligned with daily "Cost of Goods Sold" (COGS), and forecasted revenue.
+  - name: region
+    description: Region dimension table with unique region identifiers and geographic attributes.
     base_table:
       database: cortex_analyst_demo
       schema: revenue_timeseries
-      table: daily_revenue_by_region
-    time_dimensions:
-      - name: date
-        expr: date
-        description: date with measures of revenue, COGS, and forecasted revenue for each region
-        unique: false
-        data_type: date
+      table: region_dim
+    primary_key:
+      columns:
+        - region_id
     dimensions:
+      - name: region_id
+        expr: region_id
+        data_type: number
       - name: sales_region
         expr: sales_region
-        description: region associated with it's own slice of revenue
-        unique: false
+        description: Region associated with revenue
         data_type: varchar
         sample_values:
           - North America
           - Europe
           - Asia
           - South America
           - Africa
-    measures:
-      - name: daily_revenue_per_sales_region
-        expr: revenue
-        description: revenue associated with a given region for the given day
-        synonyms: ["sales", "income"]
-        default_aggregation: sum
-        data_type: number
-      - name: daily_cogs_per_sales_region
-        expr: cogs
-        description: cost of goods sold associated with a given region for the given day
-        synonyms: ["cost", "expenditures"]
-        default_aggregation: sum
-        data_type: number
-      - name: daily_forecasted_revenue_per_sales_region
-        expr: forecasted_revenue
-        description: total forecasted revenue associated with a given region for the given day
-        synonyms: ["forecasted sales", "forecasted income"]
-        default_aggregation: sum
-        data_type: number
-      - name: daily_profit_per_sales_region
-        description: profit is the difference between revenue and expenses.
-        expr: revenue - cogs
-        data_type: number
-      - name: daily_forecast_abs_error_per_sales_region
-        synonyms:
-          - absolute error
-          - L1
-        description: absolute error between forecasted and actual revenue
-        expr: abs(forecasted_revenue - revenue)
-        data_type: number
-        default_aggregation: avg
+
+  - name: product_dimension
+    base_table:
+      database: cortex_analyst_demo
+      schema: revenue_timeseries
+      table: product_dim
+
+    dimensions:
+      - name: product_line
+        expr: product_line
+        cortex_search_service_name: product_line_search_service
```

Comment:
just fyi: we are migrating to a new spec but this'll be supported as well. Okay to use this for now. Do we have flexibility to update later once the public doc moved to the new spec?